harvester/load-balancer-harvester
# Bump go mod and fix library package issue

**Description:**
Bump go mod and fix the library package issues

**Testing Steps:**
- build the image
- fix the image scanned issues
- spin up a guest rke2 cluster
- make sure the LB feature is working as usual and the LB log should not contains exceptional errors

related chart change: https://github.com/harvester/charts/pull/160
